zokrates_core/src/helpers/wasm.rs

```diff
@@ -575,47 +557,8 @@ mod tests {
             Some(ValueType::I32),
             vec![Instruction::I32Const(-1), Instruction::End],
         ));
-        let input = vec![FieldPrime::from(1)];
+        let input = vec![Bn128Field::from(1)];
         let outputs = id.execute(&input);
         assert_eq!(outputs, Err(String::from("`solve` returned error code -1")));
     }
-
-    #[test]
```

Comment:
why were these tests removed?

Reply:
Because the unpacking helper was removed as it's not generic over the field size